README.md

```diff
@@ -44,7 +44,8 @@ if 'heartbeat' in settings.INSTALLED_APPS:
 
   If you want a more detailed view of some custom checkers you MUST configure a
 custom profile for heartbeat in your settings.py. The profile should be
-a dictionary containing at least the basic auth credentials.
+a dictionary containing at least the basic auth credentials or the key `disable_auth`
+set to `True` do disable basic authentication.
```

Comment:
- The key `disable_auth` you mentioned here is not consistent with the `disable` key in the code.
- small typo after `True`: '**do** disable' instead of '**to** disable'